src/main/scala/diplomacy/LazyModule.scala

```diff
@@ -208,45 +309,85 @@ trait LazyScope
 {
   this: LazyModule =>
   override def toString: String = s"LazyScope named $name"
+  /** manage the [[LazyScope]], when calling [[apply]] function,
+    * [[LazyModule.scope]] will be altered.
+    * */
   def apply[T](body: => T) = {
-    val saved = LazyModule.scope
```

Comment:
againn why did you change the functionality here?

Reply:
Sorry, this is a mistake. I will change this.